HIRING.md

```diff
@@ -52,21 +52,32 @@ The candidate will be asked to make an assigment with a 24 hour deadline. They w
 # Job description for an iOS position
 
 ### A bit about you
-- 01 Familiarity with building applications for iOS.
-- 02 Teamplayer.
-- 03 Familiarity or interest in Swift, usability and Web services.
+- 01 You love building applications for iOS.
```

Comment:
Yeah i agree sounds a bit scary! But I do like to start with speaking directly to the reader. 

maybe 'You enjoy building applications for iOS.'